abac-common-attributes-alfa/pom.xml

```diff
@@ -0,0 +1,13 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>no.nav.abac.policies</groupId>
+        <artifactId>abac-attribute-constants</artifactId>
+        <version>4.0.0</version>
+    </parent>
+    <artifactId>abac-common-attributes-alfa</artifactId>
+    <version>4.0.4</version>
```

Comment:
This version is resolved from internal artifact while the parent pom will be incremented for each update. In addition, the project is now multi-moduled. Will that cause problems when publishing @eirslett ?